.github/workflows/test-k8s-service-buffer.yml

```diff
@@ -108,6 +109,72 @@ jobs:
         if: always()
         run: kubectl rollout status "deployment/$WORKER_RELEASE_NAME" --timeout=10s
 
+      - name: Check access to the API metrics from the DataDog Agent
+        if: always()
+        run: |
+          set -Eeuo pipefail
```

Comment:
V CI sa testuje, ze k metrikam ma pristup iba DD. Zabezpecene je to pomocou network policies nizsie.
Podobny test mame pre etcd, kde pouzivame telemetriu cez Prometheus od zaciatku.